skcosmo/selection/CUR.py

```diff
@@ -0,0 +1,218 @@
+from abc import abstractmethod
+import numpy as np
+import scipy
+from scipy.sparse.linalg import eigs as speig
+
+from ._base import _BaseSelection
+
+class _CUR(_BaseSelection):
+    """
+    Super-class defined for CUR selection methods
```

Comment:
```suggestion
    Base class for CUR selection methods
```

Why do we need a base class here?